examples/src/main/java/org/bitcoinj/examples/ForwardingService.java

```diff
@@ -1,6 +1,7 @@
 /*
  * Copyright 2013 Google Inc.
  * Copyright 2014 Andreas Schildbach
```

Comment:
Is it ok for you if we change to the generic copyright line here?

Reply:
Yeah, that's fine. I wanted to get a `2022` in there somewhere.

Reply:
Well, the generic line doesn't have a year. See 5259cc1d72ca5de0a9a296510836aa1ced9a5581.